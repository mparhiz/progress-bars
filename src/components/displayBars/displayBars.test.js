import React from 'react';
import DisplayBars from './DisplayBars';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from "enzyme-to-json";
import * as enzyme from "enzyme";

enzyme.configure({ adapter: new Adapter() });

test('renders without crashing: Enzyme', () => {
  const wrapper = enzyme.mount(<DisplayBars />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
