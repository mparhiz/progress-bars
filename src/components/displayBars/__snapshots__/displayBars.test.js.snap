// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing: Enzyme 1`] = `
<DisplayBars>
  <div
    className="bars-container"
  >
    <p>
      Loading...
    </p>
  </div>
</DisplayBars>
`;
